lib/submit-analytics.js

```diff
@@ -23,6 +23,77 @@
 
 'use strict';
 
+let utils = require('./utils');
+
+const require_method = require;
+function node_require(module) {
+    return require_method(module);
+}
+
+class webhookProdDetails {
```

Comment:
Since these classes have no state, I suggest using a more (pure) functional approach here. There's little benefit from complicating this with a class in my opinion.